org.skytemple.SkyTemple.yml

```
@@ -0,0 +1,134 @@
+app-id: org.skytemple.SkyTemple
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: run.sh
+finish-args:
+  - "--share=network"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--share=ipc"
+  - "--filesystem=host"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-nightly
+cleanup:  # todo
+  - /bin/pcap-config
+  - /include
+  - /lib/libpcap.a
+  - /lib/pkgconfig
+  - /share/man
+modules:
+  - name: armips
+    buildsystem: cmake-ninja
+    no-make-install: true
+    build-options:
+      cxxflags: "-DCMAKE_BUILD_TYPE=Release"
+    post-install:
+      - install -Dm755 armips /app/bin/armips
+    sources:
+      - type: archive
+        url: https://github.com/Kingcom/armips/archive/v0.11.0.tar.gz
+        sha256: c94e29dfda3bdf853590d825562b9c73a3d8e8e995555e021c6b2a6451572681
+      - type: patch
+        path: patches/armips.patch
+  # This needs to be done seperately due to some patches and the Rust compiler and Cargo packages required.
+  # Also this needs build isolation, unlike the rest.
+  - name: python3-skytemple-rust
+    buildsystem: simple
+    build-commands:
+      - . /usr/lib/sdk/rust-nightly/enable.sh && pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} "./skytemple-rust-1.3.7"
```

```suggestion
      - . /usr/lib/sdk/rust-nightly/enable.sh && pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} "./skytemple-rust"
```

to go with the previous change